### PR TITLE
add maintainers

### PR DESCRIPTION
**Pull request by fanux**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # Maintainers
+
+| Maintainer           | Affiliation                                                |GitHub ID | 
+| -------------------- | ------------------------------------------------------- | ----------- |
+|Haitao Fang|Alibaba|fanux
```
Comment (allencloud) [inline MAINTAINERS.md:5]: Please use Alibaba Group, rather than Alibaba

File: MAINTAINERS.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+|Hongliang Sun|Alibaba|allencloud|
+|Jiahui Huai|Boyan|bxy4543|
+|Lei Zhou|Boyan|kakaZhou719|
+|Jinrui Cui|Realai|cuisongliu|
+|Jiangnan Bao|Alibaba|justadogistaken|
```
Comment (allencloud) [inline MAINTAINERS.md:10]: I am afraid that Jiangnan has been one of Alibaba Cloud. We should find a more proper affiliation for @justadogistaken .

WDYT?

Comment (justadogistaken) [reply MAINTAINERS.md:10]: How about ZJU for now？

Comment (justadogistaken) [reply MAINTAINERS.md:10]: ZheJiang University

File: MAINTAINERS.md (hunk 1 above)
Comment (fanux) [reply MAINTAINERS.md]: done

**Review by kakaZhou719 — APPROVED** [review]

**Pull request merged**